src/sentry/templates/sentry/base-react.html

```diff
@@ -18,3 +18,18 @@
     </div>
   </div>
 {% endblock %}
+
+{% block scripts_bottom %}
+<script>
+  function setBodyTheme() {
+    if (window.__initialData.user.options.theme) {
+      document.body.classList.add(`theme-${window.__initialData.user.options.theme}`)
+    }
+  }
+
+  try {
+    setBodyTheme();
+  } catch (_) {
+  }
+</script>
+{% endblock %}
```

Comment:
Cant this be set directly on the template?

Reply:
the window.__initaldata object is injected as a jinja function that returns a string that isn't super functional to reuse. would prefer to not rewrite how we inject this context for dark mode https://github.com/getsentry/sentry/blob/master/src/sentry/templatetags/sentry_react.py#L9-L13 @BYK says he's going to give it a try

Reply:
Makes sense. I would leave a comment about this then

Reply:
https://github.com/getsentry/sentry/pull/81654

Reply:
@JonasBa comments here end up in production html since the template is not minified 😭